dataflux_pytorch/dataflux_iterable_dataset.py

```diff
@@ -130,6 +132,11 @@ def __init__(
         self.bucket_name = bucket_name
         self.data_format_fn = data_format_fn
         self.config = config
+        if not self._has_permissions():
```

Comment:
Does this check and warning still happen if the user has already disabled composed download?

Reply:
added this to the check